src/main/resources/feedbackQuestionContribResultStatsStudentViewAdditionalInfo.html

```diff
@@ -1,48 +1,45 @@
 <h4 id="interpret">How do I interpret these results? </h4>
-<ul class="bulletedList">
+<ul>
   <li>
-    Compare values given under 'My view' to those under 'Team's view'.
-    This tells you whether your perception matches with what the team thinks,
+    Compare the values given in 'My view' with those under 'Team's view'.
+    This tells you whether your perception matches what the team thinks,
     particularly regarding your own contribution. You may ignore minor differences.
   </li>
   <li>
-    Team's view of your contribution is the average value of the contribution your
-    team members attributed to you, excluding the contribution you attributed to yourself.
-    That means you cannot boost your perceived contribution by claiming a high contribution
-    for yourself or attributing a low contribution to team members.
+    From the estimates you submit, we try to deduce the answer to this question:
+    In your opinion, if your teammates are doing the project by themselves without you,
+    how do they compare against each other in terms of contribution? We
+    want to deduce your unbiased opinion about your team members’ contributions.
+  </li>
+</ul>
+<h4 id="grading">How are contribution question results used in grading?</h4>
+<ul>
+  <li>
+    TEAMMATES does not calculate grades. Instructors are free to choose how they use contribution question results.
   </li>
   <li>
-    Also note that the team’s view has been scaled up/down so that the sum of values in your
-    view matches the sum of values in team’s view. That way, you can make a direct comparison
-    between your view and the team’s view. As a result, the values you see in team’s view may
-    not match the values your team members see in their results.
+    TEAMMATES recommends that contribution question results be used only as flags to identify teams with contribution imbalances.
+    Once identified, the instructor is recommended to investigate further before taking action.
   </li>
 </ul>
-<br>
-<h4>How are these results used in grading?</h4>
-TEAMMATES does not calculate grades. It is up to the instructors to use contribution question results in any way they want.
-However, TEAMMATES recommend that contribution question results are used only as flags to identify teams with contribution imbalances.
-Once identified, the instructor is recommended to investigate further before taking action.
-<br><br>
-<h4>How are the contribution numbers calculated?</h4>
+<h4 id="calculation">How are the scores for contribution questions calculated?</h4>
+Technical details about how contribution scores are calculated can be found
+<a href="/technicalInformation.jsp#calculatePointsContribution">here</a>.
 Here are the important things to note:
```

Comment:
Sounds a little awkward to have two "here. Maybe the second one can be changed to "Below", "The following", etc.
![image](https://user-images.githubusercontent.com/26651556/37836394-4e5326dc-2eed-11e8-8b9f-f5fb556f80c5.png)